### PR TITLE
Better routing

### PR DESCRIPTION
**Pull request by evadb**
This PR looks much bigger than it actually is because I added in french placeholder copies of the pages to demo routing. 

This PR primarily adds more flexible routing to not double up the English and French routes. Routes now accept a prefix and generate the template path based on the prefix & page id. If the prefix isn't en or fr, the app throws a 404. If the route doesn't exist in general, the app still throws a 404. 

Known issues:
- Language swap button doesn't work yet, that's a future PR that builds off this one
- Organization table still links to english version of domains page, this will be fixed when I add i18n support to data tables

### CODE REVIEW

File: pulse/pulse/views.py [hunk 1]
```diff
@@ -12,22 +12,27 @@ def register(app):
...
-    def guidance():
-        return render_template("en/guidance.html")
+    @app.route("/<prefix>/feedback/")
+    def feedback(prefix):
+        return render_template(generate_path(prefix, "feedback"))
 
-    @app.route("/en/feedback/")
-    def feedback():
-        return render_template("en/feedback.html")
+    def generate_path(prefix, page_id):
```
Comment (evadb) [inline pulse/pulse/views.py]: I'm not sure if this function should live here? Seems weird to have it in the routes file, but it's a helper function specifically for the routes, and it didn't feel like it belonged in helpers.py, which is primarily for template usage. Open to moving this.

Comment (buckley-w-david) [reply pulse/pulse/views.py]: There are many options with string interpolation in python, but concatenating with `+` is generally avoided.

I would suggest doing `f'{prefix}/{page_id}.html'` instead.

File: pulse/pulse/views.py [hunk 2]
```diff
@@ -12,22 +12,27 @@ def register(app):
...
+        if(prefix == 'en' or prefix == 'fr'):
+            return prefix + "/" + page_id + ".html"
```
Comment (buckley-w-david) [inline pulse/pulse/views.py]: There are many options with string interpolation in python, but concatenating with `+` is generally avoided.

I would suggest doing `f'{prefix}/{page_id}.html'` instead.

Comment (buckley-w-david) [inline pulse/pulse/views.py]: There are many options with string interpolation in python, but concatenating with `+` is generally avoided.

I would suggest doing something like importing the `os` module, and doing
`os.path.join(prefix, f'{page_id}.html')`

Comment (evadb) [reply pulse/pulse/views.py]: right, totally forgot about that, ty!

**Pull request merged**